### PR TITLE
add npmjs search script

### PR DESCRIPTION
**Pull request by tborychowski**
## Description

Add [NPM](npmjs.com) package search script command.

## Type of change

Please delete options that are not relevant.

- [x] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Screenshot
Screenshot 2020-11-19 at 21 14 09

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by unnamedd** [comment]
@tborychowski just a reminder, both `commands/README.md` and `commands/extensions.json` are auto generated by the Raycast Toolkit, then, any change you make on those files are going to be overwritten.

File: commands/web-searches/npmjs.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title NPM search package
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon images/npmjs.png
```
Comment (unnamedd) [inline commands/web-searches/npmjs.sh:9]: You need to add the `npmjs.png` in your pull request.

File: commands/README.md [hunk 2]
```diff
@@ -206,11 +206,12 @@ This repository contains sample commands and documentation to write your own one
 | <img src="https://raw.githubusercontent.com/raycast/script-commands/master/commands/web-searches/images/google.png?raw=true" width="20" height="20"> | [Search in Google](web-searches/google-search.sh) | N/A | Raycast |
 | <img src="https://raw.githubusercontent.com/raycast/script-commands/master/commands/web-searches/images/twitter.png?raw=true" width="20" height="20"> | [Search in Twitter](web-searches/twitter-search.sh) | N/A | Raycast |
 | <img src="https://grep.app/favicon.ico" width="20" height="20"> | [Search in grep.app](web-searches/grep-app-search.sh) | N/A | Raycast |
+| <img src="https://raw.githubusercontent.com/raycast/script-commands/master/commands/web-searches/images/npmjs.png?raw=true" width="20" height="20"> | [Search in npmjs.com](web-searches/npmjs.sh) | N/A | [Tom](https://github.com/tborychowski) |
```
Comment (unnamedd) [inline commands/README.md:209]: Will be overwritten.

**Review by unnamedd — CHANGES_REQUESTED** [review]

**Comment by thomaspaulmann** [comment]
> @tborychowski just a reminder, both `commands/README.md` and `commands/extensions.json` are auto generated by the Raycast Toolkit, then, any change you make on those files are going to be overwritten.

We should mention that these files are auto-generated int the contribution guidelines. Seems like it's causing some confusion.

File: commands/web-searches/npmjs.sh (hunk 1 above)
Comment (thomaspaulmann) [reply commands/web-searches/npmjs.sh:9]: Seems like it's there now.

File: commands/web-searches/npmjs.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title NPM search package
```
Comment (thomaspaulmann) [inline commands/web-searches/npmjs.sh:5]: We prefer title-cased user-facing strings. I would suggest the following
```suggestion
# @raycast.title Search NPM Packages
```

Alternatively you could name the command just `Search Packages` and set `NPM` as `packageName`.

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Please adjust the title to follow our guidelines on title-cased userfacing strings. This way your search looks good among other commands. Thanks!

File: commands/web-searches/npmjs.sh (hunk 1 above)
Comment (unnamedd) [reply commands/web-searches/npmjs.sh:9]: Great!

File: commands/web-searches/npmjs.sh (hunk 3 above)
Comment (sindresorhus) [reply commands/web-searches/npmjs.sh:5]: `npm` should be lowercase. That's how the official name is.

Comment (tborychowski) [reply commands/web-searches/npmjs.sh]: I have no idea how to do both :-)

**Review by thomaspaulmann — APPROVED** [review]
Thanks for updating the title and great first contribution @tborychowski 🎉

File: commands/web-searches/npmjs.sh (hunk 3 above)
Comment (thomaspaulmann) [reply commands/web-searches/npmjs.sh]: Thanks @sindresorhus. I wasn't aware that this is the official name. Just updated it.

**Pull request merged**